src/plugins/plot/res/templates/mct-plot.html

```diff
@@ -217,9 +217,9 @@
                 </div>
             </div>
 
-            <div class="gl-plot-axis-area gl-plot-x">
+            <div class="gl-plot-axis-area gl-plot-x has-local-controls">
                 <mct-ticks axis="xAxis">
-                     <div ng-repeat="tick in ticks track by tick.value"
+                     <div ng-repeat="tick in ticks"
```

Comment:
Required for ngrepeat to update UI without value change.... (to my limited angular knowledge).

Reply:
This makes me nervous. Ticks are being updated at pretty high frequency, so avoiding the unnecessary regeneration of all ticks every time is desirable here. Could this be addressed by manually triggering a digest when the user changes the selected x-axis?

Reply:
Yeah. Made me nervous too. Changed it to `track by tick.text`. I cannot just do a manual digest. There are actually two digests happening when ticks are being updated... but neither of them will do anything because tick.value doesn't change.

If you don't think `tick.text`works I can so something like `${tick.value}${FORMAT_OR_TEXT_OR_WHATEVER}`.